Src/Managers/event_manager.py

```
@@ -1,81 +1,115 @@
-        _events: dict[str, list[Callable]] - словарь событий и их обработчиков
-        logger: Logger - экземпляр логгера
+@singleton
+class EventManager:
+    """
+    Менеджер событий для DearPyGui.
+    Использует ItemHandlerRegistry для событий (click, hover и т.д.).
     """
+    __logger = lateinit(logging(), 'events')
+    __global_handler_registry: str | int = None
+
+    __viewport_calls: list[Callable] = []
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)